renderer/components/Settings/Security/hooks.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+
+/**
+ * useLoginError - Sets loginError with informed form and calls `clearLoginError` afterwards
+ * to clean redux state.
+ *
+ * @param {string} loginError login error object compatible with informed api
+ * @param {string} clearLoginError action to call after error is set
+ * @param {object} formApi React ref that references informed form api
+ */
+
+export const useLoginError = (loginError, clearLoginError, formApi) => {
```

Comment:
I might be good to rename this as `useFormError` and move to global hooks dir. This could be useful for other forms that want to apply an error for a form and do something like clearing it from the state immediately after.

Reply:
good call!